### PR TITLE
[Exporter.Geneva] Remove .NET 6 target

### PR DESCRIPTION
**Pull request by joegoldman2**
## Changes

Remove .NET 6 target which will be very soon out of support.

## Merge requirement checklist

* [x] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/CONTRIBUTING.md) guidelines followed (license requirements, nullable enabled, static analysis, etc.)
* [ ] Unit tests added/updated
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project=".\Common.GenevaExporter.props"/>
 
   <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net8.0;$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj:7]: please call out this in changelog.md file, thanks!

Comment (joegoldman2) [reply src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj:7]: done

**Review by reyang — APPROVED** [review]

File: test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj [hunk 2]
```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="$(RepoRoot)\src\OpenTelemetry.Exporter.Geneva\Common.GenevaExporter.props" />
 
   <PropertyGroup>
-    <Description>Unit test project for Geneva Exporters for OpenTelemetry</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(SupportedNetTargets)</TargetFrameworks>
```
Comment (CodeBlanch) [inline test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj:7]: nit: If the goal is to standardize targets, going from `$(SupportedNetTargets)` to `net8.0` seems like a step backwards 🤷

Comment (joegoldman2) [reply test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj:7]: I propose to hardcode `net8.0` for the moment and reintroduce the property when `net6.0` will be completely removed. We can also introduce a new property temporarily now that will only contain `net8.0` if needed but not sure if it's relevant.

Comment (joegoldman2) [reply test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj:7]: I understood we don't want to impact all projects at the time and we would prefer to update them one by one. That's why I did it this way.

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Pull request merged**